packages/nextjs/pages/index.tsx

```diff
@@ -16,7 +22,7 @@ const Home: NextPage = () => {
           <p className="text-center text-lg">Get started by taking a look at your batch GitHub repository.</p>
           <p className="text-lg flex gap-2 justify-center">
             <span className="font-bold">Checked in builders count:</span>
-            <span>To Be Implemented</span>
+            <span>{ctr ? Number(ctr) : 0}</span>
```

Comment:
Two things here:

- This will show 0 when the data is loading. We should show `-` or even a spinner! There is a `isLoading` var coming from the `useScaffoldContractRead` hook that you could use.
- Doing `Number` could cause overflow, since it's a uint256. It won't happen here (not so many people are going to check in haha) but it's a good practice to use an alternative (like using a `.toString()`

Reply:
Thanks @carletex, updated the PR as per your comments. Actually I thought in that way :) (our builders number wouldnt overflow the variable) but you're right .toString() makes it flawless.